### ISSUE TITLE
Privacy Manifest

### ISSUE THREAD

**Issue by dplewis** [parse-community/Parse-SDK-iOS-OSX#1785]
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/Parse-SDK-iOS-OSX/security/policy).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/main/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/Parse-SDK-iOS-OSX/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the latest versions of [Parse Server](https://github.com/parse-community/parse-server/releases) and the [Parse ObjC SDK](https://github.com/parse-community/Parse-SDK-iOS-OSX/releases). 

### Issue Description

At WWDC23 Apple announced that apps and SDKs that make use of "required reason" APIs and "Data Collection", "Tracking" will need to provide a privacy manifest. An application will be rejected if missing a privacy manifest

https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_data_use_in_privacy_manifests
https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api

Starscream has been listed on Apple's website as a library that requires a privacy manifest. https://developer.apple.com/support/third-party-SDK-requirements

[Reminder: Privacy requirement for app submissions starts May 1](https://developer.apple.com/news/?id=pvszzano#:~:text=Starting%20May%201%2C%202024%2C%20new,Privacy%20manifests)

### Approach

* Update Starscream to 4.0.8
* Possible add Privacy Manifest to this SDK???

**Comment by dplewis** [comment]
I'm submitting an app to the App Store soon for review. I'll see if it's rejected. If it doesn't we may not need a Privacy Manifest for the SDK.

**Comment by JohnCaccavale** [comment]
I submitted an app update on Friday and was approved yesterday. Does not appear as though this SDK needs the Privacy Manifest, and is not on [the list of 3rd Party SDKs](https://developer.apple.com/support/third-party-SDK-requirements/) they are requiring Privacy Manifests for.

With that said, this issue could probably be closed.

**Comment by mtrezza** [comment]
Apple recommends all SDKs developers add a privacy manifest. It's fair to assume that the explicit list is only the beginning.

**Comment by dplewis** [comment]
@JohnCaccavale My app also got approved without issue. Can you add a privacy manifest to this SDK? Looks like you done the research.

**Comment by JohnCaccavale** [comment]
Yes, I can add in when I get a chance. Would likely be towards end of the week or weekend.

**Comment by dipeshparashar** [comment]
Is there any workaround for this. I have 4 apps with the same issue and have tried everything. Reachability seems to be the main issue here. 

![Image](https://github.com/user-attachments/assets/01f158cd-1ddb-4382-b039-92ee09d92478)
